cypress/integration/sizesAutoSpec.js

```diff
@@ -130,7 +130,8 @@ describe('On an invalid image or an image that has not loaded', () => {
           .first()
           .then(($ele) => {
             const imgSize = Number($ele.attr('sizes').split('px')[0]);
-            assert.isAtLeast(imgSize, $ele.width());
+            const expectedSize = Number($ele.attr('width').split('px')[0]);
+            assert.isAtLeast(imgSize, expectedSize);
```

Comment:
We want to test that `autoSizes` does not modify that image's `sizes` attribute. This is because an image that has not loaded should not get modified by the module.

The invalid image, `$ele`, width here will evaluate to `100vw`. This is despite the fact that it has not loaded. This is because `$ele.width()` is evaluated in the context of the test's runtime. At this point, other images on the page have loaded, changing the available width for the element. So it's not a good source of truth to ensure that `sizes` was not modified. 

Instead, we can use the elements `width` _attribute_, which does not get updated after the image above it (in the same container) loads.

Reply:
Would this be better as a code comment?

Reply:
💯, I'll add a comment and push.